sniffer.py

```diff
@@ -150,7 +156,23 @@ def main():
                 pkt = result.value[2:2+length]
                 if options.ti_crc:
                     pkt = ti_crc(pkt)
-                pkt = pcap.encode_frame(pkt)
+
+                # metadata format:
+                # 0. RSSI(int8)
+                # 1. Noise Floor(int8)
+                # 2. Flags(uint16)
+                # 3. PHY-specific data struct contains:
```

Comment:
This appears in the "vendor-specific" area as defined in the [Spinel specification](https://github.com/openthread/openthread/blob/master/doc/draft-rquattle-spinel-unified.txt).  Should we consider making these part of the standard metadata fields?